README.md

```diff
@@ -57,3 +57,12 @@ client.fetch_access_token!
 `gem install signet`
 
 Be sure `https://rubygems.org` is in your gem sources.
+
+## Supported Ruby Versions
+ This library is currently supported on Ruby 1.9+.
```

Comment:
Did you intend to have newlines instead of leading spaces here and in the next line?